src/java/org/apache/cassandra/audit/AuditLogOptions.java

```diff
@@ -37,11 +46,242 @@
     public String included_users = StringUtils.EMPTY;
     public String excluded_users = StringUtils.EMPTY;
 
-    /**
-     * AuditLogs directory can be configured using `cassandra.logdir.audit` or default is set to `cassandra.logdir` + /audit/
-     */
-    public String audit_logs_dir = System.getProperty("cassandra.logdir.audit",
-                                                      System.getProperty("cassandra.logdir",".")+"/audit/");
+    public String audit_logs_dir;
+
+    public AuditLogOptions()
+    {
+        String auditLogDir = CassandraRelevantProperties.LOG_DIR_AUDIT.getString();
```

Comment:
nit: I feel this should be `auditLogsDir` for code consistency and the below one `logsDir`. You can change it on commit if you want to

Reply:
or audit_logs_dir can be changed to `audit_log_dir` as actually the property said cassandra.logdir. I leave it to you.

Reply:
Yeah, looking around everywhere it is logDir so I suggest we change just ` audit_logs_dir` to `audit_log_dir` on commit for consistency

Reply:
Ok and in the cassandra.yaml it is audit_logs_dir..... Just leave it as is. Sorry, it was done confusing initially with difference between cassandra.logdir but in Cassandra.yaml audit_logs_dir.... 
Anyway, I am happy as is.